src/android/wifiwizard2/WifiWizard2.java

```diff
@@ -54,6 +55,8 @@
 
 import java.net.UnknownHostException;
 
+import jdk.nashorn.internal.codegen.CompilerConstants;
```

Comment:
Just realised this is in here now - possibly not required?

Reply:
It seems that this is not used, someone who uses IDE Jetbrains must have injected this.

Reply:
:raising_hand_man: Android Studio

Reply:
haha IDE